.github/pkl-workflows/modules/Steps.pkl

```diff
@@ -0,0 +1,39 @@
+import "package://pkg.pkl-lang.org/github.com/stefma/pkl-gha/com.github.action@0.0.3#/GitHubAction.pkl"
+
+checkout: GitHubAction.Step = new {
+  name = "Checkout"
+  uses = "actions/checkout@v4"
+}
+
+installPkl: GitHubAction.Step = new {
```

Comment:
This is only used once - I think it would then be better to just define it in-line in `check-actions.pkl`. Putting it here confused me thinking we uesd it multiple places in the repo, but we don't - and it's more confusing because in `ci.pkl`, we check that the "local" version of `setup-pkl` is working (ie. the same action but as defined in the source code at current checkout).

Reply:
Fixed here https://github.com/pkl-community/setup-pkl/pull/47/commits/26f6b76daa8325c3cd2a8fd8378b08f9f4d29ca3